package.json

```diff
@@ -104,7 +104,7 @@
     "copy-webpack-plugin": "^6.0.3",
     "cross-env": "^6.0.3",
     "css-loader": "^1.0.0",
-    "cypress": "^7.0.1",
+    "cypress": "8.3.1",
```

Comment:
why the strict version here, and not `^` ?

Reply:
No reason, was just copying the snippet cypress gave me